README update (#1163)
ROCm/ROCmValidationSuite: README.md

sudo zypper install rocm-validation-suite
```

## Install RVS from Tarball - for TheRock based ROCm installation

> Use the following steps to install RVS from a pre-built tarball on top of an existing ROCm Core SDK installation using TheRock build.

Reviewer: ```suggestion
Follow these steps to install RVS from a prebuilt tarball on top of an existing ROCm Core SDK installation using the TheRock build system.
```

### Download the RVS tarball

```
wget https://repo.amd.com/rocm/rvs/tarball/amdrocm7-rvs-1.4.21-288-Linux.tar.gz
```

### Extract the tarball

> Set `ROCM_PATH` to your ROCm Core SDK location. For a package manager install the default is `/opt/rocm`:

Reviewer: ```suggestion
Set `ROCM_PATH` to your ROCm Core SDK location. For a package manager installation, the default is `/opt/rocm`:
```

```
export ROCM_PATH=/opt/rocm
sudo mkdir -p $ROCM_PATH/extras-7
sudo tar -xzf amdrocm7-rvs-1.4.21-288-Linux.tar.gz -C $ROCM_PATH/extras-7
```

### Set up your environment

> **User setup** — add the following to `~/.bashrc`, then run `source ~/.bashrc`:

Reviewer: ```suggestion
**User setup** 

Add the following to `~/.bashrc`, then run `source ~/.bashrc`:
```
